web-client/src/components/ShareModal.vue

```diff
@@ -51,15 +51,15 @@ async function getIpfs(ipfsUrl, config = {}) {
 }
 
 async function getImageSrc(token) {
-    const ipfsMetaUrl = await avatarContract.getTokenUri(token)
+    const ipfsImageUrl = await avatarContract.getTokenUri(token)
 
-    if (ipfsMetaUrl) {
-        const meta = await getIpfs(ipfsMetaUrl)
+    if (ipfsImageUrl) {
+        // const meta = await getIpfs(ipfsImageUrl)
```

Comment:
A comment to prune

Reply:
Good catch 👍 we can kick that one to curbs once we've settled on my above comment about meta data :D